js/stateless.js/src/actions/create-account.ts

```diff
@@ -84,3 +89,93 @@ export async function createAccount(
 
     return txId;
 }
+
+/**
+ * Create compressed account with address and lamports
+ *
+ * @param rpc               RPC to use
+ * @param payer             Payer of the transaction and initialization fees
+ * @param seed              Seed to derive the new account address
+ * @param lamports          Number of compressed lamports to initialize the
+ *                          account with
+ * @param programId         Owner of the new account
+ * @param addressTree       Optional address tree. Defaults to a current shared
+ *                          address tree.
+ * @param addressQueue      Optional address queue. Defaults to a current shared
+ *                          address queue.
+ * @param outputStateTree   Optional output state tree. Defaults to a current
+ *                          shared state tree.
+ * @param confirmOptions    Options for confirming the transaction
+ *
+ * @return                  Transaction signature
+ */
+// TODO: add support for payer != user owner
+export async function createAccountWithLamports(
+    rpc: Rpc,
+    payer: Signer,
+    seed: Uint8Array,
+    lamports: number | BN,
+    programId: PublicKey,
+    addressTree?: PublicKey,
+    addressQueue?: PublicKey,
+    outputStateTree?: PublicKey,
+    confirmOptions?: ConfirmOptions,
+): Promise<TransactionSignature> {
+    lamports = bn(lamports);
+
+    const compressedAccounts = await rpc.getCompressedAccountsByOwner(
+        payer.publicKey,
+    );
+
+    const [inputAccounts] = selectMinCompressedSolAccountsForTransfer(
+        compressedAccounts,
+        lamports,
+    );
+
+    const { blockhash } = await rpc.getLatestBlockhash();
+
+    addressTree = addressTree ?? defaultTestStateTreeAccounts().addressTree;
+    addressQueue = addressQueue ?? defaultTestStateTreeAccounts().addressQueue;
+
+    /// TODO: enforce program-derived
+    const address = await deriveAddress(seed, addressTree);
+
+    const proof = await rpc.getValidityProof(
+        inputAccounts.map(account => bn(account.hash)),
+        [bn(address.toBytes())],
+    );
+
+    const params: NewAddressParams = {
+        seed: seed,
+        addressMerkleTreeRootIndex:
+            proof.rootIndices[proof.rootIndices.length - 1], // TODO: confirm
```

Comment:
in rust we have address Merkle tree rootIndices and state Merkle tree rootIndices.
How do you distinguish these in ts?